.github/workflows/node.js.yml

```diff
@@ -0,0 +1,21 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main, ben/ci ]
```

Comment:
Only these two branches? I made it so that it runs on all branches for `vscode-emmet-helper` here https://github.com/microsoft/vscode-emmet-helper/blob/main/.github/workflows/ci.yml

Also, I'm not sure how `pull_request` actually differs from `push`, because I notice that `push` runs per push per branch anyway, including this PR.

Reply:
Yeah I forgot to update that, pushed a change.